articles/libraries/chemistry/installation.md

```diff
@@ -50,18 +50,59 @@ For more details, see the [Package Manager Console guide](https://docs.microsoft
 dotnet add package Microsoft.Quantum.Chemistry
 ```
 
+## Verifying Your Installation 
```

Comment:
We've sometimes run into problems where users believe that running the samples is mandatory in order to use the Quantum Development Kit, which causes problems if they don't have git installed. It might help to stress that this section is optional.